notes.md

```diff
@@ -0,0 +1,30 @@
+# Notes OData POC
```

Comment:
Depening on the decision if we put the version in the request or on the entity we should polish this document to a proper decision document and put it to the `knowledge-base` folder.

Reply:
Yeah, this was more for the reviewer. I think Junjie took proper notes in the meeting.